composer.json

```diff
@@ -2,31 +2,31 @@
     "name": "ibexa/oss",
     "type": "metapackage",
     "license": "GPL-2.0-only",
```

Comment:
```suggestion
    "license": "(GPL-2.0-only or proprietary)",
```

Reply:
Thx. Updated and added LICENSE-bul via ca55b22.